app/models/badge.rb

```diff
@@ -48,7 +48,12 @@ def send_email_on_acquisition?
   # notifications when they are created
   def notification_key; end
 
-  def award_to?(_user, **_context)
+  # Stub to allow badges to short-circuit queueing
+  def self.worth_queuing?(**_context)
+    true
+  end
+
+  def award_to?(_user)
```

Comment:
```suggestion
  def award_to?(...)
```

?

Reply:
Ehm, I sort of like `_user` as it gives the reader a better indication of the interface of the method (which arguments can be expected to be passed to it)